.github/workflows/linux.yaml

```diff
@@ -29,18 +29,12 @@ jobs:
         run: bundle update
       - name: Pre-test setup
         run: |
-          echo 'runner:runner' | sudo chpasswd
-          sudo sh -c "echo 'Defaults authenticate' >> /etc/sudoers"
-          sudo sh -c "echo 'runner  ALL=(ALL) PASSWD:ALL' >> /etc/sudoers"
-          docker-compose -f spec/docker-compose.yml build --parallel ubuntu_node puppet_6_node puppet_7_node
-          docker-compose -f spec/docker-compose.yml up -d ubuntu_node puppet_6_node puppet_7_node
-          chmod 0600 spec/fixtures/keys/id_rsa
```

Comment:
We'll want to add all of these lines (except the `docker-compose` ones) to the "Pre-test setup" step for the linux integration tests.